css/style.css

```diff
@@ -1,3 +1,943 @@
-.page {
+
+/* mobile 320 */
+
+@media screen and (min-width: 320px) {
+  .page {
     font-family: "Gotham Pro", sans-serif;
+    max-width: 320px;
+    margin: 0 auto;
   }
+
+  .page__header {
+    position: relative;
+    max-width: 320px;
+    height: 50px;
+    display: flex;
+    margin: 0 auto 20px auto;
+  }
+
+  .header__logo {
+    display: none;
+  }
+
+  .header__logo-mobile {
+    position: absolute;
+    align-self: center;
+    width: 104px;
+    height: 22px;
+    left: 20px;
+    top: 24px;
+  }
+
+  .menu__left {
+    display: none;
+  }
+
+  .menu__right {
+    display: none;
+  }
+
+  .menu__burger {
+    position: absolute;
+    width: 40px;
+    height: 5px;
+    top: 28px;
+    right: 20px;
+    border-top: 6px solid #000;
+    border-bottom: 6px solid #000;
+  }
+
+  .new-collection__goto {
+    display: flex;
+    justify-content: center;
+    position: relative;
+    min-width: 100%;
+    height: 170px;
+    margin: 0 auto 30px auto;
+  }
+
+  .new-collection__goto .goto__img {
+    display: none;
+  }
+
+  .new-collection__goto .goto__img-mini {
+    width: 100%;
+  }
+
+  .new-collection__goto .goto__img-tablet {
+    display: none;
+  }
+
+  .new-collection__goto .goto__title {
+    position: absolute;
+    display: inline-block;
+    color: white;
+    font-size: 20px;
+    text-transform: uppercase;
+    text-align: justify;
+    white-space: pre-line;
+    left: 30px;
+    align-self: center;
+    line-height: 1.2;
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/eyurodin_axion/pullRequest?prid=5738041)